frigate/objects.py

```diff
@@ -82,56 +82,45 @@ def match_and_update(self, frame_time, new_objects):
             if len(current_objects) == 0:
                 for index, obj in enumerate(group):
                     self.register(index, obj)
-                return
+                continue
```

Comment:
Great find! This was probably leftover from when I updated the tracking to be grouped by label. This certainly would have caused issues with tracking when more than 1 object type was being tracked and there where frames where one object type was missed. Especially problematic for objects that are frequently confused and flip flop between cat/dog/person because object tracking is effectively disabled after the first empty group. This may be what caused this issue: ![missed](https://user-images.githubusercontent.com/54790846/118701071-51a19280-b7e1-11eb-82e2-6a07849b2dff.JPG)